person_capture/gui_app.py

```
@@ -2972,6 +3009,49 @@ def _probe_reader(c, fps_hint: float) -> bool:
+                        key="hdr_probe_deferred",
+                        interval=60.0,
+                    )
+                    ok = _probe_reader(cap, float(fps or 24.0))
+                    strict = bool(getattr(cap, "_strict_lp", False))
+                    if (not ok) and hasattr(cap, "try_fallback_chain") and not strict:
+                        try:
+                            if cap.try_fallback_chain():
+                                logger.warning("HDR reader fallback after pre-scan")
+                                ok = _probe_reader(cap, float(fps or 24.0))
+                        except Exception:
+                            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)